components/inputs/input-date.js

```diff
@@ -193,10 +197,10 @@ class InputDate extends FormElementMixin(LocalizeCoreElement(LitElement)) {
 					?label-hidden="${this.labelHidden}"
 					live="assertive"
 					@mouseup="${this._handleMouseup}"
+					offscreen-description="${ifDefined(this.emptyText ? this.emptyText : undefined)}"
```

Comment:
Is this just handling the case where `emptyText` is `''`? I would have thought `ifDefined(this.emptyText)` would have sufficed.

Reply:
It is to handle that case. I would have thought so too but sadly not.